README.md

```diff
@@ -1,7 +1,7 @@
 IITKGP ERP AUTO LOGIN
 =====================
 
-A chrome extension that you have to setup just one time, to automatically login to your IITKGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to setup your login credentials.
+You have to set up a chrome extension once, in order to login your IIT KGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to setup your login credentials.
```

Comment:
I feel here also the sense of the sentence is altered - the modification sounds like we are talking about "some extension" while the section aims at describing what IITKGP ERP AUTO LOGIN is. Can you please spare some time to look through the modification and see, if any, changes are required?